PlagiarismPlugin.php

```diff
@@ -169,88 +199,92 @@ public function callback($hookName, $args) {
 				$this->sendErrorMessage($submission->getId(), 'Could not submit "' . $submissionFile->getData('path') . '" to iThenticate.');
 			}
 		}
-
-		return false;
 	}
 
 	/**
-     * @copydoc Plugin::getActions()
-     */
-    function getActions($request, $verb) {
-        $router = $request->getRouter();
-        import('lib.pkp.classes.linkAction.request.AjaxModal');
-        return array_merge(
-                $this->getEnabled() ? array(
-            new LinkAction(
-                    'settings',
-                    new AjaxModal(
-                            $router->url($request, null, null, 'manage', null, array('verb' => 'settings', 'plugin' => $this->getName(), 'category' => 'generic')),
-                            $this->getDisplayName()
-                    ),
-                    __('manager.plugins.settings'),
-                    null
-            ),
-                ) : array(),
-                parent::getActions($request, $verb)
-        );
-    }
+	 * @copydoc Plugin::getActions()
+	 */
+	function getActions($request, $verb)
+	{
+		$router = $request->getRouter();
+		import('lib.pkp.classes.linkAction.request.AjaxModal');
```

Comment:
we can remove these `import` method calls to load class as we have proper namespace feature now